deploy/crds/policy.open-cluster-management.io_operatorpolicies.yaml

```diff
@@ -39,24 +42,29 @@ spec:
           metadata:
             type: object
           spec:
-            description: OperatorPolicySpec defines the desired state of OperatorPolicy
+            description: OperatorPolicySpec defines the desired state of a particular
+              operator on the cluster.
             properties:
               complianceType:
-                description: ComplianceType describes whether we must or must not
-                  have a given resource
+                description: |-
+                  ComplianceType describes how objects on the cluster should be compared with the object definition
+                  of the configuration policy. The supported options are "MustHave", "MustOnlyHave", or
+                  "MustNotHave".
```

Comment:
Similar suggestion about quotation mark consistency. I recommend that we use the backtick